frontend/packages/dev-console/integration-tests/reporter-config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "reporterEnabled": "mocha-junit-reporter, mochawesome",
+    "mochaJunitReporterReporterOptions": {
+      "mochaFile": "../../../gui_test_screenshots/junit_cypress-[hash].xml",
```

Comment:
Hmm...hope this `[hash]` will be unique between admin and dev-console tests, since the files will be written to the same place.